Readme.md

```diff
@@ -85,6 +85,12 @@ Finally, create a event, to stop the music when we unlock the phone
 
 Exit Tasker with the exit menu to be sure all events and task have been saved.
 
+Steps to get your personal Key:
+
+    Go to your personal URL
```

Comment:
Indentation here means "code highlight" in markdown.
Please remove the indentation as it's not code here.